README.md

```diff
@@ -15,13 +15,57 @@ The structure of this repository is the following:
   - `subspace-farmer` is a CLI farmer app
 - `substrate` contains modified copies of Substrate's crates that we use for testing
 
-### How to run
+### How to run (building from the source)
 
 This is a monorepo with multiple binaries and the workflow is typical for Rust projects:
 
 - `cargo run --release --bin subspace-node -- --dev --tmp` to run [a node](crates/subspace-node)
 - `cargo run --release --bin subspace-farmer -- farm` to [start farming](crates/subspace-farmer#start-the-farmer)
 
-NOTE: You need to have `nightly` version of Rust toolchain with `wasm32-unknown-unknown` target available or else you'll get a compilation error.
+NOTE 1: You need to have `nightly` version of Rust toolchain with `wasm32-unknown-unknown` target available or else you'll get a compilation error.
+NOTE 2: Following the commands above, you will be farming in an offline setting (by yourself).
 
 You can find readme files in corresponding crates for requirements, multi-node setup and other details.
+
+
+## Releases (executables)
+
+We are regularly releasing stable snapshots. Our CI builds container images and executables for 3 major platforms (Windows, MacOS, Linux). 
+You can find these executables in the [Releases](https://github.com/subspace/subspace/releases).
+With the provided executables, you can choose to `Farm` online by joining the network, or offline (by yourself, for test or development purposes).
+
+### To Farm With The Network (Online)
+
+**Mac/Linux**
+
+1. Download the executables for your operating system
+2. Open your favourite terminal, and go to the folder where you download the executables
+3. This will start the node: `sudo ./subspace-node-x86_64-*-snapshot-test \
```

Comment:
1) NO SUDO!!!
2) File needs to be made executable after download, by default it will not be in most cases

Reply:
Is Chmod +x okay?

Reply:
Yes